spec/requests/admin_claims_spec.rb

```diff
@@ -20,10 +20,23 @@
   describe "claims#show" do
     let(:claim) { create(:claim, :submitted) }
 
-    it "returns a claim when one exists" do
-      get admin_claim_path(claim)
+    context "when no other claim has matching attributes" do
```

Comment:
I think it would be ok to leave this `it` block as it is. Wrapping it in the additional context just makes it look a bit strange and not really make sense without knowledge of the context that comes after it. Whereas the context after makes sense on it's own without the need to have this it in an opposite context.

Reply:
Agreed; I'll change it.